ciao-launcher/main.go

```diff
@@ -241,17 +241,20 @@ func (client *agentClient) installLauncherDeps(doneCh chan struct{}) {
 
 	ch := make(chan error)
 	go func() {
-
 		osprepare.Bootstrap(ctx)
 
+		launcherDeps := make(osprepare.PackageRequirements)
```

Comment:
This line is a little confusing.  At first I didn't think it was valid go until I realised that PackageRequirements was a map.  It might be better to create a constructor function for this type.  This way launcher doesn't need to know what the underlying implementation is.  Having said that I can't find any best practices document suggesting that we shouldn't do this.  So feel free to ignore if you disagree.